examples/vanilla/media-elements/mux-video.html

```diff
@@ -8,10 +8,15 @@
       src="https://cdn.jsdelivr.net/npm/@mux/mux-video/+esm"
```

Comment:
NOTE: Updating example to demo some of the more advanced crud. Useful for smoke testing, too.